playbooks/clean-bigip-partitions.yml

```diff
@@ -132,4 +135,44 @@
         label: "{{ item.name }}"
       when: item.name != "Common"
 
-# TODO: clear certificates and kinds of profiles under Common
\ No newline at end of file
+    - name: remove client ssl profiles
+      bigip_profile_client_ssl:
+        name: "{{ item.name }}"
+        state: absent
+        provider:
+          user: admin
+          server: "{{ inventory_hostname }}"
+          password: "{{ admin_password }}"
+          validate_certs: False 
+      with_items: "{{ bigip_infos.client_ssl_profiles }}"
+      loop_control:
+        label: "{{ item.name }}"
+      when: (item.name | regex_search('^' + environment_prefix + '[\d\w\s\D\W\S]+'))
```

Comment:
Client ssl profiles' name format is uuidv4. Please replace `[\d\w\s\D\W\S]+` with `_[0-9a-f\-]{36}` for the matching.
Same situation to another 2 `[\d\w\s\D\W\S]+`.